_includes/footer.html

```diff
@@ -1,31 +1,10 @@
 <footer class="site-footer h-card">
-    <data class="u-url" href="{{ " / " | relative_url }}"></data>
-
+    <data class="u-url" href="{{ "/" | relative_url }}"></data>
     <div class="wrapper">
-
-        <div class="footer-col-wrapper">
-            <div class="footer-col one-half">
-                <h2 class="footer-heading">{{ site.title | escape }}</h2>
-                <ul class="contact-list">
-                    <li class="p-name">
-                        {%- if site.author -%} {{ site.author | escape }} {%- endif -%}
-                    </li>
-                    {%- if site.email -%}
-                    <li><a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a></li>
-                    {%- endif -%}
-                </ul>
-            </div>
-
-            <div class="footer-col one-half">
-                <p>{{- site.description | escape -}}</p>
-            </div>
-
-            <div class="social-links">
-                {%- include social.html -%}
-            </div>
-        </div>
-
         <div class="footer-col-wrapper">
+            <p style="text-align: center">
+                <img src="/assets/images/logos/aerosec.svg" alt="logo" />
+            </p>
```

Comment:
It makes more sense to me to put the aerosec logo at the top of the page, rather than the bottom. But feel free to do that in a different PR.

Reply:
Moved logo to top, revamped title.